divide.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+
+def divide(a, b):
+    type1 = type(a)
+    type2 = type(b)
+
+    if a is None:
+        return b
+
+    if b is None:
+        return a
+
+    if type1 is not int:
+        raise TypeError('First input parameter must be an int. Instead it is ' + str(type1))
+
+    if type2 is not int:
+        raise TypeError('Second input parameter must be an int. Instead it is ' + str(type2))
```

Comment:
This code is a duplicate of what is in the add method. Please centralize them in a check_type method as we discussed in the class to keep the code DRY. Please see the sample code in my pull request - https://github.com/vidyaschools/Calculator-Batch1/pull/6/commits/d850a52ff03f443dfcb0c391a630a5b0831928a9 (Note: I have closed my pull request since it didn't contain the unit tests for some of the methods I had added.)